spyder_terminal/widgets/style/themes.py

```diff
@@ -0,0 +1,387 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright (c) Spyder Project Contributors
+#
+# Licensed under the terms of the MIT License
+# (see LICENSE.txt for details)
+# -----------------------------------------------------------------------------
+ANSI_COLORS = {
```

Comment:
Question. The values for these colors come from the theme?

Some of them? all of them?

Reply:
None of them, the colors that I'm using from the configuration are in the `set_theme` method. This colors were created using a generator for terminal themes depending the ones that are in spyder.

Reply:
> This colors were created using a generator for terminal themes depending the ones that are in spyder.

Is that generator a Python library? Or how could we use it in the future?

Reply:
Is an online generator, https://terminal.sexy/, which allows you to create a terminal color scheme that makes sense. But it is manual, so each time a new theme is added to spyder we need to create the ANSI colors for it manually.